### ISSUE TITLE
Unable to create/edit product - PG::UndefinedColumn

### ISSUE THREAD

**Issue by jjf21** [solidusio-contrib/solidus_globalize#55]
**Steps to reproduce**
1. Create a new rails app, add gems : 

```
gem 'solidus'
gem 'solidus_auth_devise'
gem 'globalize', github: 'globalize/globalize'
gem 'solidus_i18n', github: 'solidusio-contrib/solidus_i18n'
gem 'solidus_globalize', github: 'solidusio-contrib/solidus_globalize'
```

2. Generate migrations
```
bundle exec rails g spree:install
bundle exec rails g solidus:auth:install
bundle exec rake railties:install:migrations
bundle exec rake db:migrate
bin/rails g solidus_globalize:install
bin/rails g solidus_i18n:install
```

3. Go to admin/product/product.
Add update or create a product

**Expected behavior**
Product should be save in the DB

**Actual behavior**
Error 500 : 
`PG::UndefinedColumn - ERROR:  column friendly_id_slugs.locale does not exist`

**System configuration**
* Rails 5.1.3
* Solidus 2.3.0
* Solidus i18n 1.1.0
* Solidus Globalize 3.1.0.beta

**Comment by bofrede** [comment]
I have the same problem in Production, and got the same error yesterday on Rails 5.1.4 in an application created from scratch.
Tried both SQLite3 and MySQL, but to no avail.
Product editing did work before I installed Solidus Globalize.

**Comment by bofrede** [comment]
I finally got it working. :-)

Add this migration:
``` ruby
class AddLocaleToFriendlyIdSlugs < ActiveRecord::Migration[5.0]
  def change
    add_column :friendly_id_slugs, :locale, :string
  end
end
```

**Comment by kennyadsl** [comment]
Fixed in #101.  Sorry for the long response time.
